### ISSUE TITLE
How to change the encryption key, method, algorithm, etc..

### ISSUE THREAD

**Issue by m-primo** [elgibor-solution/laravel-database-encryption#22]
How to change method, key, and edit the algorithm.
I think I need to publish the package and do custom edit myself, but there are no publish resources for this package.
Any thoughts?

**Comment by charliefortune** [comment]
@m-primo Did you find any solution to this? I'm looking for the same - specifically to set my own encryption key.

**Comment by m-primo** [comment]
> @m-primo Did you find any solution to this? I'm looking for the same - specifically to set my own encryption key.

No, I didn't, but according to this package's code, it uses `APP_KEY` environment variable to set the encryption key, you can edit but, but I don't really think it's a good idea, that's why I was asking if there's a way to publish it, like the config at least.
This is the config file: [/src/Config/config.php](https://github.com/elgiborsolution/laravel-database-encryption/blob/main/src/Config/config.php).

I noticed a user mentioning ( #13  [https://github.com/elgiborsolution/laravel-database-encryption/issues/13#issuecomment-1049183297](https://github.com/elgiborsolution/laravel-database-encryption/issues/13#issuecomment-1049183297) ) that we can copy the config.php file into another file and change the keys, but I haven't tried it, and I don't know which file name should it be, maybe as is config.php? I don't really know.

**Comment by charliefortune** [comment]
> I noticed a user mentioning ( #13 [#13 (comment)](https://github.com/elgiborsolution/laravel-database-encryption/issues/13#issuecomment-1049183297) ) that we can copy the config.php file into another file and change the keys, but I haven't tried it, and I don't know which file name should it be, maybe as is config.php? I don't really know.

That's useful thanks, seems like it could work. I would give it a different name (e.g. encryption.php) and save it in the config dir, I assume it gets autoloaded in there. I'll give it a try and update here if it works.

I agree that sharing/changing the APP_KEY is a bad idea. It's not even an option for me in my project, really.
